samples/azure-functions/service-bus-worker/sample.md

```diff
@@ -15,21 +15,19 @@ downloadbutton
 
 ## Prerequisites
 
-### Manually create queue
+### Create queue
 
-Unlike a traditional NServiceBus endpoint, an endpoint hosted in Azure Functions cannot create its own input queue; the queue must be created elsewhere, for example in the Azure portal or with the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli). In this sample, the queue name is `ASBWorkerEndpoint`.
+The use the sample the endpoint queue must be created. It can be done manually in the Azure portal, or with the [Azure CLI](https://docs.microsoft.com/en-us/cli/azure/install-azure-cli). In this sample, the queue name is `ASBWorkerEndpoint`.
 
 To create the queue with the Azure CLI, execute the following Azure CLI command:
 
 ```
 az servicebus queue create --name ASBWorkerEndpoint --namespace-name <asb-namespace-to-use> --resource-group <resource-group-containing-namespace>
 ```
 
-### Manually create bundle topic
-
-```
-az servicebus topic create --name bundle-1 --namespace-name <asb-namespace-to-use> --resource-group <resource-group-containing-namespace>
-```
+#if-version [4.0,)
```

Comment:
In version 4 onwards we fixed the installers so that they do actually create the queues/topics